resources/hab_build.rb

```diff
@@ -13,6 +13,7 @@
 property :home_dir, String
 property :auth_token, String
 property :live_stream, [TrueClass, FalseClass], default: true
+property :cleanup, [TrueClass, FalseClass], default: false
```

Comment:
It doesn't say how the values are compared to the type (ie, `is_a?` vs `===`), but it shows `true` and `false` as example types in the docs: https://docs.chef.io/custom_resources.html#property